### ISSUE TITLE
[Bug][A11Y] Switch: error status not read by Talkback and Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#365]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S22 and iPhone 14+

### OS Version

Android and iOS

### App version

0.5.0 and 0.6.0

### Describe the issue

For both OS, the "error" status is not read by screen-readers

### Expected Behavior

Error status shall be read

**Comment by nouha06** [comment]
@pya35 The error status should be read by screen redear do you mean in the main component or customization  or both ?

**Comment by pya35** [comment]
> [@pya35](https://github.com/pya35) The error status should be read by screen redear do you mean in the main component or customization or both ?

@nouha06 I mean in the main component, this is correctly done on checkbox component but not on switch

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (alpha) (410)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 410
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (410)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 410
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by pya35** [comment]
ok

**Comment by B3nz01d** [comment]
closed during sprint # 22
